src/test/resources/features/delete_account.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Account
+
+  Background:
+    Given I sign in as "owner"
+    And I Create an Account "XXXGP"
+
+  Scenario: Delete Account
+    When I Delete the Account "XXXGP"
+    Then I validate "XXXGP" account was successfully deleted message is displayed
+    Then I validate "XXXGP" account is not displayed in Accounts menu
+
```

Comment:
It is missing validation in account dropdown in create project and account dropdown in project setting form.